### PR TITLE
Section2 Complete

### PR DESCRIPTION
**Pull request by Eystein-Thanisch**

### CODE REVIEW

File: Rawl_B502/genelach_úa_Fáeláin.ttl [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+<#Finn>
+    a foaf:Person;
+    irishRel:nomName "Finn";
+    irishRel:genName "Finn";
+    rel:childOf <#MáelMórdai>;
+    owl:sameAs <#Find>.
+
+<#MáelMórdai>
+    a foaf:Person;
+    irishRel:nomName ;
```
Comment (cyocum) [inline Rawl_B502/genelach_úa_Fáeláin.ttl]: I think this should be irishRel:nomName "Máel Mórda" but please make sure I am correct :)

Comment (Eystein-Thanisch) [inline Rawl_B502/genelach_úa_Fáeláin.ttl]: Yup. I agree. Now corrected.

File: Rawl_B502/geneleach_osrithe.ttl [hunk 2]
```diff
@@ -0,0 +1,841 @@
...
+<#Amalgada>
+    a foaf:Person;
+    irishRel:nomName "Amalgaid";
+    irishRel:genName "Amalgada";
+    rel:childOf <#LóegaireBirnBuadaig>;
+    owl:sameAs <http://example.com/LL/rig_ossairge.ttl#Amalgaid>.
+
+<#LóegaireBirnBuadaig>
+    a foaf:Person;
+    irishRel:nomName Lóegaire Birn Buadaig;
```
Comment (cyocum) [inline Rawl_B502/geneleach_osrithe.ttl]: Should this be "Lóegaire Birn Buadach"?

**Comment by Eystein-Thanisch** [comment]
Yup, indeed it should. Corrected.

**Comment by cyocum** [comment]
Hi,

Thanks.  Have you pushed the changes you made to GitHub? I cannot seem them
yet.

Chris

**Comment by Eystein-Thanisch** [comment]
It turns out I hadn't pressed "Sync". They should be there now?

**Comment by cyocum** [comment]
Yep, looks good.  Thanks.

**Pull request merged**